do.html

```diff
@@ -83,15 +83,17 @@ <h3 class="main-sub-header">What are you waiting for?<br> Just <strong>Do</stron
 <p>
   We strongly believe in the benefits of modular design,
   collaboration, and re-use. That is why instead of developing a
-  single monolithic platform to "solve all the problems", we are
-  trying to reuse and contribute as much as possible to existing
-  projects. To that end, in the scope of this project we actively
-  maintain relevant software
-  within <a href="http://neuro.debian.net">NeuroDebian</a> turnkey
-  platform, so you could benefit from new solutions such
-  as <a href="http://singularity.lbl.gov">Singularity</a> to make your
-  research more flexible and reproducible. With our developments we
-  aim to provide you with a collection of tools useful on their own
+  single monolithic platform to "solve all the problems", we are as
+  much as possible reuse and contribute to existing projects. To that
+  end, in the scope of this project we actively maintain relevant
+  software within <a href="http://neuro.debian.net">NeuroDebian</a>,
+  so that ReproNim and other projects could benefit from this turnkey
+  platform.  We also provided official Debian packaging and assist in
+  maintaining
```

Comment:
I think this sentence should be rephrased, but I'm afraid my attempt in af5604988 lost some information.